services/mesh-networking/pkg/traffic-policy-temp/translation/translators/istio/translator.go

```
@@ -46,6 +47,29 @@ func (i *istioTrafficPolicyTranslator) Name() string {
 	return TranslatorId
 }
 
+// mutate the translated snapshot, adding the translation results in where appropriate
+func (i *istioTrafficPolicyTranslator) AccumulateFromTranslation(
+	snapshotInProgress *snapshot.TranslatedSnapshot,
+	meshService *smh_discovery.MeshService,
+	allMeshServices []*smh_discovery.MeshService,
+	mesh *smh_discovery.Mesh,
+) error {
+	if snapshotInProgress.Istio == nil {
+		snapshotInProgress.Istio = &snapshot.IstioSnapshot{}
+	}
+
+	out, errs := i.Translate(meshService, allMeshServices, mesh, meshService.Status.ValidatedTrafficPolicies)
+
+	snapshotInProgress.Istio.DestinationRules = append(snapshotInProgress.Istio.DestinationRules, out.DestinationRules...)
+	snapshotInProgress.Istio.VirtualServices = append(snapshotInProgress.Istio.VirtualServices, out.VirtualServices...)
+	// assuming all went well in the previous stages, there should be no errors
+	if len(errs) != 0 {
+		// ?? panic ??
```

Return an error here just in case something goes wrong

---

that's actually something that i wanted to chat with you - IIUC the correct behaviour here is to ignore the error